learning_observer/learning_observer/rosters.py

```diff
@@ -409,10 +413,18 @@ async def courselist(request):
     '''
     List all of the courses a teacher manages: Helper
     '''
-    # New code
-    if settings.pmss_settings.source(types=['roster_data']) in ["google_api"]:
-        runtime = learning_observer.runtime.Runtime(request)
-        return await learning_observer.google.courses(runtime)
+    
+    # A map of LMSes to their respective handler functions
+    lms_map = {
+        "google_api": learning_observer.google.courses,
```

Comment:
This isn't something you need to worry about, but I think we ought to rename the roster source `google_api` to `google` to be more consistent across the system.

Reply:
Changed and also made it a constant too